dev-lang/ruby/ruby-3.2.4-r3.ebuild

```diff
@@ -3,7 +3,9 @@
 
 EAPI=8
 
-inherit autotools flag-o-matic multiprocessing
+RUST_OPTIONAL="yes"
+
+inherit autotools flag-o-matic multiprocessing rust
```

Comment:
How does RUST_OPTIONAL work with pkg_setup? I don't see a pkg_setup() { use jit && ... } or anything in this ebuild?